docs/public/SUMMARY.md

```diff
@@ -0,0 +1,110 @@
+- [Installation]()
+    - [Prerequisites](installation/prerequisites.md)
+    - [Zonemaster-LDNS](installation/zonemaster-ldns.md)
+    - [Zonemaster-Engine](installation/zonemaster-engine.md)
+    - [Zonemaster-CLI](installation/zonemaster-cli.md)
+    - [Zonemaster-Backend](installation/zonemaster-backend.md)
+    - [Zonemaster-GUI](installation/zonemaster-gui.md)
+    - [Docker](installation/docker.md)
+- [Upgrade]()
+    - [Backend](upgrade/backend.md)
+- [Configuration]()
+    - [Profiles](configuration/profiles.md)
+    - [Backend](configuration/backend.md)
+        - [Environment Variables](configuration/backend-environment-variables.md)
+    - [GUI](configuration/gui.md)
+- [Using]()
+    - [CLI](using/cli.md)
+    - [RPCAPI](using/getting-started.md)
```

Comment:
This file may be better named `backend-rpcapi-getting-started.md`. Having this file named just `getting-started` could lead to confusion as it is not a general "Getting started" but the "Getting started" of the RPCAPI.

Reply:
Why not just "backend-getting-started.md"?